spec/factories/profile_fields.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :profile_field do
+    label { "Email" }
```

Comment:
Since we are validating uniqueness for this field I think we should use an incrementer to set the value like we do for emails and such to ensure these records are always valid for tests.

Reply:
Good idea, done.